tests/test_password.py

```diff
@@ -55,8 +55,8 @@ def test_failed_reset_password_empty_email(self, cli_runner):
         email = ""
```

Comment:
I wouldn't be testing this at all and leave it for Elis API to response. Because like this you should also check whether the email is really an existing email address, if it's an email address, if the user is using string sign or not... you can remove this test.